Cargo.toml

```diff
@@ -18,12 +18,6 @@ authors = [
   "Philipp Schuster <phip1611@gmail.com>"
 ]
 
-[profile.release]
-codegen-units = 1
-opt-level = "s"
-lto = true
```

Comment:
> Removing LTO from the build options cuts build times in half for no apparent downside.

Here are my measurements:
- 1.82.0-nightly + existing codegen options : 20,91s
- 1.82.0-nightly + default codegen options  : 11,39s
- 1.80.1 (stable) + existing codegen options: 20,77s
- 1.80.1 (stable) + default codegen options : 11.85s

I agree that there is a benefit. Further, I verified that `opt-level = "s"` only saves 300K. As the micro optimization really isn't that important and it apparently solves problems for you, I'm fine with the changes.